test/integration/tests/abrmd_policysigned.sh

```diff
@@ -102,4 +102,34 @@ tpm2 flushcontext session.ctx
 diff secret.dat unsealed.dat
 rm -f unsealed.dat
 
-exit 0
+#
+# Test with cpHashA with ECDSA signature
```

Comment:
Suggestion: Test policy-signing with ECDSA signing scheme and cpHashA included in the message.